test_cases.md

```diff
@@ -0,0 +1,67 @@
+# Budget Test Smoke Suite.
+
+Test Suite consists of unit and functional tests.
+Unit tests are located in <code>test_exp_calc_unittest.py</code> file
+They are covering individual functions used by Click commands.
+Click commands are tests are located in <code>test_exp_calc_pytest.py</code>
```

Comment:
Click commands functional tests are located in <code>test_exp_calc_pytest.py</code>